### PR TITLE
Umbraco.StorageProviders.AzureBlob version 9 info 

### PR DESCRIPTION
**Pull request by RachBreeze**
Version 1.1.1 is the correct version of Umbraco.StorageProviders.AzureBlob for v9 of Umbraco

Spent a while on this when tracking down a build failure hope this helps

### CODE REVIEW

File: Extending/FileSystemProviders/Azure-Blob-Storage/index-v9.md [hunk 1]
```diff
@@ -17,15 +17,15 @@ See the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/storage
...
+1. Use your favorite IDE and open up the NuGet Package Manager to search and install version 1.1.1 of the package
+1. Use the command line to install the package using `Install-Package Umbraco.StorageProviders.AzureBlob -Version 1.1.1`
 
 ### Installing through command line
 
 Navigate to your project folder, which is the folder that contains your `.csproj` file. Now use the following `dotnet add package` command to install the package:
 
 ```none
-dotnet add package Umbraco.StorageProviders.AzureBlob
+dotnet add package Umbraco.StorageProviders.AzureBlob -v 1.1.1'
```
Comment (eshanrnh) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index-v9.md:28]:
```suggestion
dotnet add package Umbraco.StorageProviders.AzureBlob --version 1.1.1
```

**Review by eshanrnh — COMMENTED** [review]
Thanks for the PR, @RachBreeze 🙌 This is quite helpful 😃

I've added a small suggestion.

**Comment by RachBreeze** [comment]
Hi @eshanrnh thank you I do like that change

**Review by eshanrnh — APPROVED** [review]
Looks good 🙌 Merging it in.

**Pull request merged**